parsers/NA.py

```diff
@@ -52,6 +50,31 @@ def get_text_from_image(link, expected_size, new_size, logger, session=None):
     return text
 
 
+def timestamp_checker(session=None, logger=None):
```

Comment:
very small detail, maybe rename to `check_timestamp`? I'd expect a "Checker" to be a class more than a function